src/validate_oNHM_daily_output.R

```diff
@@ -34,9 +32,26 @@ validate_oNHM_daily_output <- function(var, fn, test_date, data_nc, hruids, time
   assert_that(dim(data_nc) == n_hrus) 
   
   # General order of magnitude test
-  # Max of total storage in all of historic data for all HRUs is ~5,000 mm
+  # Max of total storage in all of historic data for all HRUs is ~8,000 mm
   # Considering 300 mm is about 1 ft of water
   # An absolute max for order of magnitude check of 10,000 mm seems appropriate
-  assert_that(all(data_nc < 10000))
+  data_is_good <- all(data_nc < 10000)
+  if(data_is_good) {
+    message("DATA = GOOD")
+    # Write out a text file that won't have Jenkins send an email.
+    # Add information about the current var to the file.
+    write(x = sprintf("%s data: NULL", var), 
```

Comment:
Since this is appending, you might want to first check if `order_magnitude_test.txt` exists or not and delete it if so, in case there is workspace contamination from older jobs.  I think this shouldn't be an issue, but just in case.

Reply:
Hmmm I didn't think about contamination from older jobs. Just made the assumption that it would be wiped each time. Will add a check